www/secmail/views/forms/icla.js.rb

```diff
@@ -89,6 +89,7 @@ def componentDidMount()
     end
 
     @realname = name
+    @pubname = name
```

Comment:
Just so we are clear on what this change does: this will pre-fill in the public name with the name of the person who sent in the form.  So if John Smith from Oracle sends in 15 pdfs with ICLAs for NetBeans, you will have to overwrite John Smith 30 times.  What the current logic does is fill in the name only once, allowing you to change it only once.  Tabbing to the next field will get a copy of the name, which you should only have to change if the public name is different than the full name.  Before I merge this change, please confirm that this is the behavior you want.  And, if so, the "onFocus: -> {@pubname ||= @realname}" can be removed earlier in this code.